Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Time Switch/Time Switch](https://hosted.weblate.org/projects/timeswitch/timeswitch/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/timeswitch/-/timeswitch/horizontal-auto.svg)

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/

fsobolev committed:
Translated using Weblate (Russian)

Currently translated at 100.0% (73 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/ru/

flipflop97 committed:
Translated using Weblate (Dutch)

Currently translated at 95.8% (70 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/nl/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/

flipflop97 committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (73 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/nl/

flipflop97 committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (73 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/nl/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/

fsobolev commented:
@gregorni please update translation when you will have time, a new release is planned for the end of next week :)

fsobolev committed:
Translated using Weblate (Russian)

Currently translated at 100.0% (73 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/ru/

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (73 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/de/

ghost commented:
Everything done on my end!

DeltaVeintiuno committed:
Translated using Weblate (Spanish)

Currently translated at 68.4% (50 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/es/

DeltaVeintiuno committed:
Translated using Weblate (Spanish)

Currently translated at 97.2% (71 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/es/

eson57 committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (73 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/sv/

DeltaVeintiuno committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (73 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/es/

plasmus777 committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (73 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/pt_BR/

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (73 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/uk/

fsobolev committed:
Translated using Weblate (Russian)

Currently translated at 100.0% (73 of 73 strings)

Translation: Time Switch/Time Switch
Translate-URL: https://hosted.weblate.org/projects/timeswitch/timeswitch/ru/
